packages/components/src/button/style.scss

```diff
@@ -194,7 +194,41 @@
 
 	// Link buttons that are red to indicate destructive behavior.
 	&.is-link.is-destructive {
-		color: $alert-red;
+		color: #b52727;
+	}
+
+	/**
+	 * 	Destructive buttons.
```

Comment:
Nicely commented. I think we should use a single space instead of a tab between the asterisk and comment, i.e. `* Destructive...`.